t/format-datetime.t

```diff
@@ -0,0 +1,20 @@
+use strict;
+use warnings;
+
+use Test::More 0.96;
+use Test::Fatal;
+
+use DateTime::Format::Strptime;
+use DateTime;
+
+my $strptime = DateTime::Format::Strptime->new(
+    pattern  => '%B %Y',
+    locale   => 'pt',
+    on_error => 'croak',
+);
+
+
+eval { $strptime->format_datetime('somestring') };
```

Comment:
If you're using the Test::Fatal module, you could combine the `eval { ... }` and `like( $@, ... )` into `like( exception { ... }, ... )`.